download_wasm.sh

```diff
@@ -7,30 +7,30 @@ rm -rf src/pkg/findex
 rm -rf src/pkg/cover_crypt
 
 SOURCE=${BASH_SOURCE[0]}
-DIR=$( cd -P "$( dirname "$SOURCE" )" >/dev/null 2>&1 && pwd )
+DIR=$(cd -P "$(dirname "$SOURCE")" >/dev/null 2>&1 && pwd)
 
 if [[ -z "${CI_JOB_TOKEN}" ]]; then
     if [[ -d "../findex" ]]; then
-        pushd ../findex
+        pushd ../findex || exit
     else
         git clone git@gitlab.cosmian.com:core/findex.git /tmp/findex
-        pushd /tmp/findex
+        pushd /tmp/findex || exit
     fi
 
     git checkout $FINDEX_VERSION
-    wasm-pack build --target web -d $DIR/src/pkg/findex --release --features wasm_bindgen
-    popd
-    
+    wasm-pack build --target web -d "$DIR/src/pkg/findex" --release --features wasm_bindgen
+    popd || exit
+
     if [[ -d "../cover_crypt" ]]; then
-        pushd ../cover_crypt
+        pushd ../cover_crypt || exit
     else
         git clone https://github.com/Cosmian/cover_crypt.git /tmp/cover_crypt
-        pushd /tmp/cover_crypt
+        pushd /tmp/cover_crypt || exit
     fi
 
     git checkout $COVER_CRYPT_VERSION
-    wasm-pack build --target web -d $DIR/src/pkg/cover_crypt --release --features wasm_bindgen
-    popd
+    wasm-pack build --target web -d "$DIR/src/pkg/cover_crypt" --release --features wasm_bindgen
+    popd || exit
```

Comment:
I think we can set `set -euxo pipefail` at the beginning of the file instead of this. See https://www.howtogeek.com/782514/how-to-use-set-and-pipefail-in-bash-scripts-on-linux